.jenkins/jenkins.sh

```diff
@@ -53,9 +53,10 @@ shopt -s expand_aliases
 # setup module environment and default queue
 test -f ${envloc}/env/machineEnvironment.sh || exitError 1201 ${LINENO} "cannot find machineEnvironment.sh script"
 . ${envloc}/env/machineEnvironment.sh
-
+export python_env=${python_env}
+echo "PYTHON env ${python_env}"
 # get root directory of where jenkins.sh is sitting
-root=`dirname $0`
+export root=`dirname $0`
```

Comment:
is it not confusing to you to call the `.jenkins` directory root? In other scripts we use the root as root and I think it would increase readability if we called this `scriptdir` or something along those lines

Reply:
Thank you so much for catching this, yes totally you are right, this is confusing